build.sh

```diff
@@ -35,9 +38,23 @@ function fetch_icu {
 
 ICU_TOOLS_ROOT=`pwd`/icu/source
 ICU_LLVM_ROOT=`pwd`/icu-llvm/source
-export HOST_ARG="--host=x86_64-apple-darwin"
 
-export CXXFLAGS="${CFLAGS} -fvisibility-inlines-hidden -stdlib=libc++ -std=c++11"
+
+UNAME=`uname -s`
+if [ ${UNAME} = 'Darwin' ]; then
+    export HOST_ARG="--host=x86_64-apple-darwin"
+    CONFIG_NAME="mh-darwin"
+    IN_PLACE="-i 'bak'";
+	STDLIB="stdlib=libc++";
```

Comment:
I think this should be `STDLIB="-stdlib=libc++";` @ChrisLoer rather than `STDLIB="stdlib=libc++";`